adhocracy-plus/config/urls.py

```diff
@@ -67,9 +65,6 @@
 ct_router.register(r'comment-moderate', CommentModerateSet,
                    basename='comment-moderate')
 
-question_router = QuestionDefaultRouter()
-question_router.register(r'vote', VoteViewSet, basename='vote')
-
 urlpatterns = [
```

Comment:
We will also need to remove that in Opin and MeinBerlin, I will prepare pull requests, that we can then merge once we update a4 there..